controllers/transactionController.js

```diff
@@ -0,0 +1,98 @@
+//Controller for items
+import Transaction from "../model/schemas/Transaction.js";
+import db from "../model/db.js";
+
+const transactionController = {
+    // The transaction page
+    transactions: function (req, res) {
+        res.render("transactions", {
+            title: "Transactions",
+            styles: ["index.css", "w2ui-overrides.css"],
+            scripts: ["transaction.js"],
+        });
+    },
+
+    addTransaction: async function (req, res) {
+        var transItem = {
+            date: req.body.date,
+            type: req.body.type,
+            description: req.body.desc,
+            quantity: req.body.qty,
+            sellingPrice: req.body.sellingPrice,
+            transactedBy: req.body.transactedBy
+        }
+
+        console.log('Added transaction: ')
+        console.log(transItem)
+
+        db.insertOne(Transaction, transItem, function (data) {
+            res.send(data);
+        });
+    },
+
+    getTransactions: function (req, res) {
```

Comment:
Add `try...catch`
In `catch`, return `500` with `message: Server error`

Reply:
see [cd1c739](https://github.com/erik-lance/TitafindsXG5/pull/11/commits/cd1c739754a0c62c4b2346b66ee0e497f871eb46)